templates/containers/what-are-containers.html

```diff
@@ -14,282 +14,250 @@
+        <p>
+          CRI (Container Runtime Interface) is a Kubernetes API to interface between Kubernetes orchestration features and container runtimes, such as containerd or CRI-O. The <a href="https://opencontainers.org/">Open Container Initiative</a> (OCI) defines a set of specifications to enable all these technologies to work together. Docker, runc, containerd, and CRI-O are all OCI compatible.
         </p>
-      </div>
-      <div class="col-5 u-align--center u-vertically-center u-hide--medium u-hide--small">
-        {{ image(url="https://assets.ubuntu.com/v1/cd384737-Kubernetes+v+Docker.svg",
-                alt="",
-                width="300",
-                height="105",
-                hi_def=True,
-                loading="lazy") | safe
-        }}
       </div>
     </div>
   </section>
 
-  <section class="p-strip u-no-padding--top u-no-padding--bottom">
-    <div class="p-strip--suru-topped">
-      <div class="row u-equal-height">
-        <div class="col-7 u-vertically-center">
-          <p>
-            Choosing the right Kubernetes distribution is crucial for your business.
-            <br />
-            <a href="/engage/enterprise-kubernetes-comparison">Read the Kubernetes distribution comparison whitepaper&nbsp;&rsaquo;</a>
-          </p>
-        </div>
-        <div class="col-5 u-align--center u-vertically-center u-hide--medium u-hide--small">
-          {{ image(url="https://assets.ubuntu.com/v1/198b738b-Compare_Kubernetes-Openshift-Rancher.svg",
-                    alt="",
-                    width="240",
-                    height="150",
-                    hi_def=True,
-                    loading="lazy") | safe
-          }}
+  <section class="p-section u-fixed-width">
+    <hr class="p-rule" />
+    <div class="u-fixed-width">
+      <div class="p-logo-section">
+        <div class="p-logo-section__items">
+          <div class="p-logo-section__item">
+            <img class="p-logo-section__logo"
+                 src="https://assets.ubuntu.com/v1/59fd3874-k8s-docker.png"
+                 alt="" />
+          </div>
         </div>
       </div>
     </div>
+    <div class="p-section u-fixed-width u-fixed-width">
+      <p class="p-heading--2">
+        Not clear on how Kubernetes vs Docker differ?
+        <br />
+        <a href="/blog/kubernetes-versus-docker">Read the Kubernetes vs Docker blog&nbsp;&rsaquo;</a>
+      </p>
+    </div>
   </section>
 
-  <section class="p-strip--light">
-    <div class="u-fixed-width">
-      <h2>Containers vs virtual machines</h2>
-      <p>
-        Containers are a modern way to virtualise infrastructure, more lightweight than traditional virtual machines: all containers on a single host OS share the kernel and other resources, require less memory space, ensure greater resource utilisation and shorter startup times by orders of magnitude.
+  <section class="u-fixed-width">
+    <hr class="p-rule" />
+    <div class="p-strip is-deep u-fixed-width">
```

Comment:
```suggestion
  <section class="u-fixed-width">
    <hr class="p-rule" />
    <div class="p-strip is-deep">
```
Not needed since this is already wrapped in a row via the first `u-fixed-width`